extensions/docsearch/src/algolia/APIData.ts

```diff
@@ -0,0 +1,178 @@
+import type { IAPIData } from "./types";
```

Comment:
for the sake of consistency it would make more sense to rename this file to "apiData.ts"